todo_app/data/session_items.py

```diff
@@ -12,15 +12,11 @@ def get_items():
 
 
 def get_item(id):
-    #Fetches the saved item with the specified ID. Args:id: The ID of the item.
-    #Returns: item: The saved item, or None if no items match the specified ID.
```

Comment:
Why are we editing this file? It's unused, so we'd be better off just deleting it

Reply:
deleted